src/azure-cli/HISTORY.rst

```diff
@@ -117,6 +117,7 @@ Release History
 
 **Redis**
 
+* Fixing `az redis update` operation to work for caches with RDB/AOF enabled
```

Comment:
- Is there any GitHub issue related to this PR?
If yes, please add issue link in PR description.
- Please also refine the PR title to <type>[optional scope]: <description> to make title more clear for others, you can reference to [https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary)

Reply:
> Is there any GitHub issue related to this PR?
> If yes, please add issue link in PR description.

There is no GitHub issue. Issue was found with support ticket from customer.
> Please also refine the PR title, for examples, "fix:Trimming RDB/AOF connection string in redis cache update operation" to make title more clear for others, you can reference to https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary

Updated.